src/components/RadioConditionalInput.tsx

```diff
@@ -204,31 +204,25 @@ const renderConditionalDateInputs = (
             className={`govuk-radios__conditional ${error ? '' : 'govuk-radios__conditional--hidden'}`}
             id={radio.dataAriaControls}
         >
-            <fieldset className="govuk-fieldset" role="group" aria-describedby={radio.hint.id}>
-                <legend className="govuk-fieldset__legend govuk-fieldset__legend--s" id={radio.hint.id}>
-                    {radio.hint.content}
-                </legend>
-                {radio.inputs.map(input => {
-                    const inputGroupError = radio.inputErrors.find(({ id }) => {
-                        return id.includes(input.id);
-                    });
+            {radio.inputs.map(input => {
+                const inputGroupError = radio.inputErrors.find(({ id }) => {
+                    return id.includes(input.id);
+                });
 
-                    const dayValue = input.name === 'startDate' ? dates.startDateDay : dates.endDateDay;
-                    const monthValue = input.name === 'startDate' ? dates.startDateMonth : dates.endDateMonth;
-                    const yearValue = input.name === 'startDate' ? dates.startDateYear : dates.endDateYear;
+                const dayValue = input.name === 'startDate' ? dates.startDateDay : dates.endDateDay;
+                const monthValue = input.name === 'startDate' ? dates.startDateMonth : dates.endDateMonth;
+                const yearValue = input.name === 'startDate' ? dates.startDateYear : dates.endDateYear;
 
-                    return (
+                return (
+                    <fieldset className="govuk-fieldset" role="group">
```

Comment:
We should wrap this in a `<div className="govuk-form-group">` as well to get the spacing right